model/utils.go

```diff
@@ -587,3 +588,29 @@ func GetPreferredTimezone(timezone StringMap) string {
 
 	return timezone["manualTimezone"]
 }
+
+type NullBool struct {
+	sql.NullBool
+}
+
+func (nb NullBool) MarshalJSON() ([]byte, error) {
+	if nb.Valid {
+		return json.Marshal(nb.Bool)
+	} else {
+		return json.Marshal(nil)
```

Comment:
I just realized that `nil` should really be marshalled as `false` because semantically that's what it means. The nil value only exists because we wanted to avoid the downtime of a migration.

Reply:
Makes sense, I thought that we wanted to distinguish between `false` and `nil`. Patched